Lib/sqlite3/__main__.py

```diff
@@ -0,0 +1,98 @@
+import sqlite3
+import sys
+
+from argparse import ArgumentParser
+from code import InteractiveConsole
+from textwrap import dedent
+
+
+def execute(c, sql):
+    try:
+        for row in c.execute(sql):
+            print(row)
+    except sqlite3.Error as e:
+        tp = type(e).__name__
+        try:
+            print(f"{tp} ({e.sqlite_errorname}): {e}")
+        except:
```

Comment:
```suggestion
        except Exception:
```
Or use a narrow exception.

Reply:
Oops, I intended to use attribute error here. I'll fix that; thanks!